src/mono/System.Private.CoreLib/src/System/Runtime/InteropServices/MemoryMarshal.Mono.cs

```diff
@@ -8,6 +8,45 @@ namespace System.Runtime.InteropServices
 {
     public static unsafe partial class MemoryMarshal
     {
+
+        /// <summary>
+        /// Creates a new span over a portion of a regular managed object. This can be useful
+        /// if part of a managed object represents a "fixed array." This is dangerous because the
+        /// <paramref name="length"/> is not checked.
+        /// </summary>
+        /// <param name="reference">A reference to data.</param>
+        /// <param name="length">The number of <typeparamref name="T"/> elements the memory contains.</param>
+        /// <returns>A span representing the specified reference and length.</returns>
+        /// <remarks>
+        /// This method should be used with caution. It is dangerous because the length argument is not checked.
+        /// Even though the ref is annotated as scoped, it will be stored into the returned span, and the lifetime
+        /// of the returned span will not be validated for safety, even by span-aware languages.
+        /// </remarks>
+        [Intrinsic]
+        [NonVersionable]
```

Comment:
IIRC, Mono does not care about either of these attributes so this should not be needed.

Reply:
Ah ok, I'll test that later when I'm back home.